### ISSUE TITLE
Unable to find a match: dotnet-sdk-6.0 for RHEL

### ISSUE THREAD

**Issue by Swimburger** [dotnet/sdk#26431]
### Describe the bug
I am following [these instruction](https://access.redhat.com/documentation/en-us/net/6.0/html/getting_started_with_.net_on_rhel_8/installing-dotnet_getting-started-with-dotnet-on-rhel-8) to try and install the .NET 6 SDK on RHEL 8.
When I try to run the installation command, I get an error. This could be because I'm not familiar with RHEL and I am missing some obvious step that's not part of this guide.
The RHEL docs do have a link "Performing a standard RHEL installation" that points to a 404, and I can't find it, so maybe the missing step is hidden in that document 🤷.

Also, using the command document in the [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/core/install/linux-rhel#rhel-8-), also doesn't work.

### To Reproduce
- Create a RHEL VM, I created one using Azure, RHEL 8.2 in my case.
- Try to run `sudo yum install dotnet-sdk-6.0 -y` to install the .NET 6 SDK

Microsoft Docs command: `sudo dnf install dotnet-sdk-6.0` 

### Exceptions (if any)
The error I'm getting from the `yum` command is the following.
```
Updating Subscription Management repositories.
This system is registered to Red Hat Subscription Management, but is not receiving updates. You can use subscription-manager to assign subscriptions.
Last metadata expiration check: 0:10:07 ago on Mon 04 Jul 2022 08:42:31 PM UTC.
No match for argument: dotnet-sdk-6.0
Error: Unable to find a match: dotnet-sdk-6.0
```

The `dnf` command error:
```
Updating Subscription Management repositories.
This system is registered to Red Hat Subscription Management, but is not receiving updates. You can use subscription-manager to assign subscriptions.
Last metadata expiration check: 2:32:39 ago on Mon 04 Jul 2022 08:42:31 PM UTC.
No match for argument: dotnet-sdk-6.0
Error: Unable to find a match: dotnet-sdk-6.0
```

**Comment by Swimburger** [comment]
@richlander suggested I include @omajid into the conversation

**Comment by omajid** [comment]
Hi @Swimburger

.NET 6 was a new addition to RHEL 8.5: https://www.redhat.com/en/blog/whats-new-rhel-85. It should be available in 8.5 and later, but not in 8.4 or earlier. Can you upgrade from 8.2 to to 8.5 or later? If so, you should be able to `dnf install` .NET 6.

**Comment by Swimburger** [comment]
Learning how to upgrade was its own challenge (for a newbie).
After upgrading `sudo yum install dotnet-sdk-6.0 -y` and `sudo dnf install dotnet-sdk-6.0` worked!
